src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Blocks/tabs.html.twig

```diff
@@ -42,11 +43,18 @@
         {{ 'Pricing'|trans({}, 'Admin.Catalog.Feature') }}
       </a>
     </li>
+
     <li id="shipping-tab-nav" class="nav-item{% if not isShippingTabValid %} has-error{% endif %}">
       <a href="#shipping-tab" role="tab" data-toggle="tab" class="nav-link">
         {{ 'Shipping'|trans({}, 'Admin.Catalog.Feature') }}
       </a>
     </li>
+
+    <li id="options-tab-nav" class="nav-item{% if not isOptionsTabValid %} has-error{% endif %}">
+      <a href="#options-tab" role="tab" data-toggle="tab" class="nav-link">
+        {{ 'Options'|trans({}, 'Admin.Catalog.Feature') }}
```

Comment:
This strings already exists in Admin.Global
```suggestion
        {{ 'Options'|trans({}, 'Admin.Global') }}
```

Reply:
same as above, please confirm once again that I should use Global (previous product page seems to use Admin.Catalog.Feature)

Reply:
Yep, I confirm :) Thank you very much

Reply:
Are you sure about this @Julievrz ? `Options` is quite a common word so no doubt it is in the global namespace but this one is dedicated to a Product tab in the Product edition page So the context is more specific, it is possible that even though the wording is the same in english in some other languages it was not translated exactly the same way because of the context

Reply:
Oh my bad, in the current page the global namespace is already used indeed https://github.com/PrestaShop/PrestaShop/blob/fb29bfbb9f9aa773cbe45b4211c7115898c8c881/src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/Blocks/tabs.html.twig#L36

@zuk3975 I think we can rely on the legacy page when we have doubts about wording or namespaces Most of them are here https://github.com/PrestaShop/PrestaShop/tree/develop/src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage